src/main/knn/KnnGraphTester.java

```diff
@@ -776,7 +785,7 @@ private void testSearch(Path indexPath, Path queryPath, int queryStartIndex, Pat
     Result[] results = new Result[numQueryVectors];
     int[][] resultIds = new int[numQueryVectors][];
     long elapsed, totalCpuTimeMS, totalVisited = 0;
-    ExecutorService executorService = Executors.newFixedThreadPool(8);
```

Comment:
we had this ExecutorService already? What were we using it for -- was this used for indexing maybe, or for computing brute force matches? If so, we should keep this one as-is and make a new one to use as the searcher threadpool

Reply:
It was unused. I think it was left over from previous experiments. You can see that it wasn't used anywhere else.